src/components/Panel/Panel.js

```diff
@@ -4,17 +4,34 @@ import cx from 'classnames';
 import Template from '../Template/Template';
 
 class Panel extends Component {
+  state = {
+    collapsed: this.props.collapsed,
+    controlled: false,
+  };
+
+  static getDerivedStateFromProps(nextProps, prevState) {
```

Comment:
It's only available since `8.3.0` of `preact`. We should at least require this version.